runelite-client/src/main/java/net/runelite/client/plugins/interfacestyles/InterfaceStylesPlugin.java

```diff
@@ -209,7 +209,7 @@ private SpritePixels getFileSpritePixels(String file, String subfolder)
 		return null;
 	}
 
-	private SpritePixels getImageSpritePixels(BufferedImage image)
+	public static SpritePixels getImageSpritePixels(BufferedImage image, Client client)
```

Comment:
Don't reference this from another plugin

Reply:
So its better to copy and paste the exact code?

Reply:
`banktags.tabs.TabSprites` has the same method. They would be moved to something like `ImageUtils`